frontend/src/utils/mdxPageHelpers.ts

```diff
@@ -45,7 +45,7 @@ export function getMDXContentBySlug(
   slug: string,
   locale = "en",
 ): MDXData {
-  const itemPath = path.join(`${pathBase}/${slug}/${locale}.mdx`);
+  const itemPath = path.join(`${pathBase}/${slug}/${(locale === "bo") ? "en" : locale}.mdx`);
```

Comment:
This is a hack to force english static pages when tibetan is selected as a UI language. The fallbackLng option for i18n seems to be when a language is selected that is not supported at all and we have partial support for Tibetan. However, I might be completely wrong here! So remove if not okay.